src/siwe-client.ts

```diff
@@ -1,21 +1,21 @@
 import { AuthRequestBody, ClockResponse } from '@fiatconnect/fiatconnect-types'
 import { ethers } from 'ethers'
 import { generateNonce, SiweMessage } from 'siwe'
-import { CookieJar, MemoryCookieStore } from 'tough-cookie'
 import {
   ClockDiffParams,
   ClockDiffResult,
   SiweApiClient,
   SiweClientConfig,
   SiweLoginParams,
+  CookieJarType,
 } from './types'
 
 export class SiweImpl implements SiweApiClient {
   config: SiweClientConfig
   signingFunction: (message: string) => Promise<string>
   fetchImpl: typeof fetch
   _sessionExpiry?: Date
-  cookieJar: CookieJar
+  _loginHeader: Headers
```

Comment:
nit: could just make this optional similar to _sessionExpiry and update node's getCookies to use optional chaining